libvast/vast/system/reader_command.hpp

```diff
@@ -106,10 +140,14 @@ caf::message reader_command(const command& cmd, caf::actor_system& sys,
       return caf::make_message(std::move(in.error()));
     Reader reader{slice_type, std::move(*in)};
     VAST_INFO(reader, "reads data from", *file);
-    auto src = sys.spawn(source<Reader>, std::move(reader), factory, slice_size,
-                         max_events);
-    return source_command(cmd, sys, std::move(src), options, first, last);
+    src = sys.spawn(source<Reader>, std::move(reader), factory, slice_size,
+                    max_events);
+  }
+  if (schema) {
+    caf::scoped_actor self{sys};
```

Comment:
What's the rationale for spinning up a `scoped_actor` here? Would `anon_send` perhaps suffice?